ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/event-handlers.j2

```diff
@@ -52,21 +49,22 @@ event-handler {{ handler.name }}
 {%                 if handler.trigger_on_logging.regex is arista.avd.defined %}
       regex {{ handler.trigger_on_logging.regex }}
 {%                 endif %}
-{%             elif handler.trigger is arista.avd.defined("on-intf") and handler.trigger_on_intf.interface is arista.avd.defined %}
-{%                 if handler.trigger_on_intf.ip is arista.avd.defined(true) or
-                     handler.trigger_on_intf.ipv6 is arista.avd.defined(true) or
-                     handler.trigger_on_intf.operstatus is arista.avd.defined(true) %}
-{%                     set trigger_on_intf_cli = "trigger on-intf " ~ handler.trigger_on_intf.interface %}
-{%                     if handler.trigger_on_intf.operstatus is arista.avd.defined(true) %}
-{%                         set trigger_on_intf_cli = trigger_on_intf_cli ~ " operstatus" %}
-{%                     endif %}
-{%                     if handler.trigger_on_intf.ip is arista.avd.defined(true) %}
-{%                         set trigger_on_intf_cli = trigger_on_intf_cli ~ " ip" %}
-{%                     endif %}
-{%                     if handler.trigger_on_intf.ipv6 is arista.avd.defined(true) %}
-{%                         set trigger_on_intf_cli = trigger_on_intf_cli ~ " ip6" %}
-{%                     endif %}
+{%             elif handler.trigger is arista.avd.defined("on-intf") %}
+{%                 if handler.trigger_on_intf.interface is arista.avd.defined %}
+{%                     set trigger = [] %}
+{%                     for intf_change in ['operstatus', 'ip', 'ipv6'] %}
+{%                         if handler.trigger_on_intf[intf_change] is arista.avd.defined(true) %}
+{%                             if intf_change == "ipv6" %}
+{%                                 do trigger.append("ip6") %}
+{%                             else %}
+{%                                 do trigger.append(intf_change) %}
+{%                             endif %}
+{%                         endif %}
+{%                     endfor %}
+{%                     if trigger %}
+{%                         set trigger_on_intf_cli = "trigger on-intf " ~ handler.trigger_on_intf.interface ~ " " ~ trigger | join(" ") %}
```

Comment:
It's not really becoming more readable with this. And not saving any lines...

Reply:
Okay, I have reverted the changes to if/else